tests/karma/js-unit.conf.js

```diff
@@ -97,11 +97,12 @@ module.exports = function (config) {
 			'extensions/wikia/AdEngine/js/EventDispatcher.js',
 			'extensions/wikia/AdEngine/js/EvolveHelper.js',
 			'extensions/wikia/AdEngine/js/EvolveSlotConfig.js',
-			'extensions/wikia/AdEngine/js/lookup/*.js',
 			'extensions/wikia/AdEngine/js/MessageListener.js',
 			'extensions/wikia/AdEngine/js/WikiaDartHelper.js',
 			'extensions/wikia/AdEngine/js/WikiaDartVideoHelper.js',
 			'extensions/wikia/AdEngine/js/WikiaGptAdDetect.js',
+			'extensions/wikia/AdEngine/js/config/*.js',
+			'extensions/wikia/AdEngine/js/lookup/*.js',
```

Comment:
It is alphabetical: letters A-Z come before a-z, right? ;-)

On 04/03/2015 11:44 AM, Andrzej wrote:
> In tests/karma/js-unit.conf.js
> <https://github.com/Wikia/app/pull/6823#discussion_r27722679>:
>
>>  			'extensions/wikia/AdEngine/js/MessageListener.js',
>>  			'extensions/wikia/AdEngine/js/WikiaDartHelper.js',
>>  			'extensions/wikia/AdEngine/js/WikiaDartVideoHelper.js',
>>  			'extensions/wikia/AdEngine/js/WikiaGptAdDetect.js',
>> +			'extensions/wikia/AdEngine/js/config/*.js',
>> +			'extensions/wikia/AdEngine/js/lookup/*.js',
>
> It's an interesting way of ordering: alphabetical but within directory
> :japanese_ogre: I think it can stay like that because the whole fire
> seems to have "sub-directories sections".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6823/files#r27722679>.
>